autogpt_platform/backend/backend/api/features/briefings/routes_test.py

```
@@ -0,0 +1,165 @@
+"""Tests for the briefings API routes.
+
+Pattern mirrors backend/api/features/executions/review/review_routes_test.py:
+the full app from backend.api.rest_api, the session-scoped `server` fixture
+(real Prisma DB), and `mock_jwt_user` for auth overrides. Unlike the review
+tests, these seed real rows via backend.data.briefing.create_briefing rather
+than mocking the data layer, since the behavior under test includes how the
+route handles content that fails to validate against BriefingContent.
+"""
+
+import datetime
+from typing import AsyncGenerator
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)